content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -95,10 +95,10 @@ At this point, the command has two additional side effects. The command installs
 
 ## 3. Generate Prisma Client for the Data Proxy
 
-Next, generate Prisma Client for the Data Proxy with the `--data-proxy` flag. Later, you use the [Data Proxy](/data-platform/classic-projects/data-proxy) to connect to your database over HTTP.
+Next, generate Prisma Client for the Data Proxy with the `--no-engines` flag. Later, you use the [Data Proxy](/data-platform/classic-projects/data-proxy) to connect to your database over HTTP.
```

Comment:
Do we want to start recommending Accelerate over Data Proxy for our deployment guides that require its usage?

Reply:
Yes, but switching to Data Proxy here would cause updates to the rest of the guide which I didn't want to do as a part of this PR.